bin/wallet-history

```diff
@@ -4,39 +4,96 @@ var _ = require('lodash');
 var fs = require('fs');
 var moment = require('moment');
 var program = require('commander');
+var async = require('async');
 var utils = require('./cli-utils');
 program = utils.configureCommander(program);
 
+
+program
+  .option('-o, --output <file>', 'get JSON output in file')
+  .option('-s, --skip <n>', 'skip the first n transactions (0 by default)')
+  .option('-p, --page <n>', 'download page size')
+  .option('-l, --limit <n>', 'limit download to n transactions')
+  .option('-i, --info', 'include extended info');
+
 program
   .parse(process.argv);
 
 var args = program.args;
 
-utils.getClient(program, { mustExist: true }, function (client) {
-  client.getTxHistory({}, function (err, txs) {
-    if (_.isEmpty(txs))
-    return;
-
-    console.log("* TX History:")
-
-    _.each(txs, function(tx) {
-      var time = moment(tx.time * 1000).fromNow();
-      var amount = utils.renderAmount(tx.amount);
-      var confirmations = tx.confirmations || 0;
-      var proposal = tx.proposalId ? '["' + tx.message + '" by ' + tx.creatorName + '] ' : '';
-      switch (tx.action) {
-        case 'received':
-          direction = '<=';
-        break;
-        case 'moved':
-          direction = '==';
-        break;
-        case 'sent':
-          direction = '=>';
-        break;
-      }
+var skip = +program.skip || 0,
+  page = +program.page || 55,
```

Comment:
55 is the 10th Fibonacci number and a triangular number (the sum of the consecutive numbers 1 to 10). It is the largest Fibonacci number to also be a triangular number.